specs/HubProtocol.md

```diff
@@ -489,13 +516,40 @@ is decoded as follows:
 
 * `0x94` - 4-element array
 * `0x03` - `3` (Message Type - `Result` message)
-* `0xa3` - string of length 3 (Target)
+* `0xa3` - string of length 3 (InvocationId)
 * `0x78` - `x`
 * `0x79` - `y`
 * `0x7a` - `z`
 * `0x03` - `3` (ResultKind - Non-Void result)
 * `0x2a` - `42` (Result)
 
+### CancelInvocation Message Encoding
+
+`CancelInvocation` messages have the following structure
+
+```
+[4, InvocationId]
+```
+
+* `4` - Message Type - `4` indicates this is a `CancelInvocation` message
```

Comment:
I need 4 for StreamCompletion

Reply:
You should start registering these with IANA :trollface:

Reply:
I registered with @BrennanConroy last week :)